prover/strategies/duplicate.md

```diff
@@ -0,0 +1,33 @@
+# duplicate
+
+This strategy is useful in combination with strategies that modify
+the proof context.
+```
+
+Gamma, Phi, Phi |- Psi
+-----------------
+Gamma, Phi |- Psi
+
+```
```

Comment:
Fix formatting: Put new lines before and after the paragraphs, instead of inside the code blocks